modules/cli/src/main/scala/scala/cli/exportCmd/Json.scala

```diff
@@ -0,0 +1,186 @@
+package scala.cli.exportCmd
+
+import com.github.plokhotnyuk.jsoniter_scala.core.WriterConfig
+import coursier.ivy.IvyRepository
+import coursier.maven.MavenRepository
+import coursier.parse.RepositoryParser
+import coursier.{LocalRepositories, Repositories}
+import dependency.NoAttributes
+
+import java.nio.charset.StandardCharsets
+import java.nio.file.Path
+
+import scala.build.internal.Constants
+import scala.build.internal.Runner.frameworkName
+import scala.build.options.{BuildOptions, Platform, ScalaJsOptions, ScalaNativeOptions, Scope}
+import scala.build.testrunner.AsmTestRunner
+import scala.build.{Logger, Sources}
+import scala.cli.util.SeqHelpers.*
+
+final case class Json(
```

Comment:
NIT I feel a more descriptive name would be better, `Json` is too general and doesn't really indicate what this class is for...
Maybe smth like:
```suggestion
final case class JsonProjectDescriptor(
```

Reply:
same applies to the `Mill` and `Sbt` classes, so you can rename them consistently as well...